### ISSUE TITLE
Can't get 90.01 result given by the paper

### ISSUE THREAD

**Issue by None403** [zxlzr/FewShotNLP#4]
Hello，friend！
    In the process of my reproduction, I used bert-base-uncased as the pre-training model and ran the ARSC_pretrain.sh. In the end, I only achieved an accuracy of 86.1, but did not obtain the 90.01 result given in the paper.
When I use bert-large-uncased, the result is only 21.83.
Therefore, I have the following two questions:
1. How did you achieve 90.01, whether you use bert base or bert large, is there a super parameter corresponding to bert large that needs to be updated?
2. What do I need to do for my 90.01 result?
result with bert-large-uncased
![image](https://user-images.githubusercontent.com/24206257/79530411-d3386580-80a1-11ea-8653-bb80fe9a1175.png)
result with bert-base-uncased

![image](https://user-images.githubusercontent.com/24206257/79539016-0c7bd000-80b8-11ea-8f89-7ad8509c6759.png)

**Comment by zxlzr** [comment]
Thank you for your comment. We forget to introduce some preprocessing procedures.  We fond there are some wrong labels in the dataset so we filter and correct those training data by rules.  We also replace some long  OOV words with synonyms. But we do not think those are the key reasons!

We have also conducted experiments multiple times and we found we also cannot get that results (we only get the best results 89.9 and 90.01  twice with different random seeds). We can get the 0.87 with BERT-base and 0.88-0.89 with Roberta. After careful analysis, we think there are two main reasons. 
1. There exits a high variance in fine-tuning BERT.   We train the model with a normal initial learning rate and then train the model with SGD with a very low learning rate. Surprisingly we get better results. Maybe you can try this.  I think you may get overfitting with BERT-large. 
2 Catastrophic forgetting in further pertaining. We find we get extremely poor performance sometimes.  We think the model forgets the linguistic knowledge obtained from original pretraining. We use a low initial learning rate and it works.  

To conclude, I hope you may get better results with those tricks.   Moreover, a larger batch size will improve performance.   But the high variance and different seeds really have a big influence.  I hope future researchers can address those issues.

**Comment by None403** [comment]
Thank you for your reply！！！
Can you provide a pre-processing script for the data you mentioned above?

**Comment by Wangpeiyi9979** [comment]
can you share your preprocessing procedures？

**Comment by VivianYWY** [comment]
> Hello，friend！
> In the process of my reproduction, I used bert-base-uncased as the pre-training model and ran the ARSC_pretrain.sh. In the end, I only achieved an accuracy of 86.1, but did not obtain the 90.01 result given in the paper.
> When I use bert-large-uncased, the result is only 21.83.
> Therefore, I have the following two questions:
> 
> 1. How did you achieve 90.01, whether you use bert base or bert large, is there a super parameter corresponding to bert large that needs to be updated?
> 2. What do I need to do for my 90.01 result?
>    result with bert-large-uncased
>    ![image](https://user-images.githubusercontent.com/24206257/79530411-d3386580-80a1-11ea-8653-bb80fe9a1175.png)
>    result with bert-base-uncased
> 
> ![image](https://user-images.githubusercontent.com/24206257/79539016-0c7bd000-80b8-11ea-8f89-7ad8509c6759.png)

hello, i also want to reproduce this code, but i can not find Amazon _corpus.txt in the folder data. could you please send me a copy ? thanks a lot
